openbts/components.py

```diff
@@ -263,6 +263,21 @@ def update_port(self, imsi, new_port):
     }
     return self._send_and_receive(message)
 
+  def get_imsi_from_number(self, number):
```

Comment:
Nit: can you add some doc about the return type here? e.g., "returns IMSI as string if a subscriber exists w/ that number; raises an InvalidRequestError if number doesn't exist."